src/constants.js

```diff
@@ -8,13 +8,18 @@ import {
   ExternalLinkAltIcon,
   OutlinedQuestionCircleIcon,
 } from '@patternfly/react-icons';
+import {
+  Text,
+  TextContent,
+} from '@patternfly/react-core/dist/js/components/Text';
```

Comment:
Why not just use `'@patternfly/react-core'`?

Reply:
I thought we were optimizing our imports by importing form the specific file.